dockerfiles/alpine/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM rust:1.72 as builder
+
+ENV HOME=/home/root
+WORKDIR $HOME/app
+# rust toolchain
+ARG RUST_TC_VER=stable
+ARG BUILD_TARGET=x86_64-unknown-linux-musl
+ARG BUILD_PROFILE=release-lto
+
+# import the Nodesource GPG key
+RUN apt-get update \
```

Comment:
Probably, it would be better to build backend and frontend separately. Backend can use `rust` image and frontend - `node` image. In this way there will be no need to write script that installs nodejs and yarn